cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -173,6 +173,12 @@ type VolumeConfigFlags struct {
 	PersistentVolumeRecyclerIncrementTimeoutHostPath    int
 }
 
+const (
+	DefaultNodeMonitorGracePeriod = 40 * time.Second
+	DefaultNodeStartupGracePeriod = 60 * time.Second
+	DefaultNodeMonitorPeriod      = 5 * time.Second
```

Comment:
Can you put these with all the other defaults (in NewCMServer, then reference them with s.NodeMonitorGracePeriod)?